### PR TITLE
cc

### PR DESCRIPTION
**Pull request by jpcosal**

### CODE REVIEW

**Comment by thomas83520** [comment]
Dsl mais c'était pas fou

File: Exam/src/exercice4/Bilan.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package exercice4;
+
+import java.util.Date;
+
+public class Bilan
```
Comment (thomas83520) [inline Exam/src/exercice4/Bilan.java]: oh oui c'est bien

File: Exam/src/exercice2/LogItemTest.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+class LogItemTest {
+
+	@Test
+	void testLogItem() throws Exception
+	{
+		LogItem li = new LogItem(new Date(), "MASTER", "Here is a simple message");
+		
+		String disp = li.format();
+		
+		System.out.println(disp);
```
Comment (jpcosal) [inline Exam/src/exercice2/LogItemTest.java]: Where is the assert clause??

File: Exam/src/exercice3/TestJouet.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+		
+		Forme[] compJouet = {c, t, r};
+		Jouet jouet = new Jouet(compJouet);
+		double aire = jouet.calculateArea();
+		
+		System.out.println(aire);
+		//aire = 16.641592653589793
+		
+		
+		Locale currentLocale = Locale.getDefault();
```
Comment (jpcosal) [inline Exam/src/exercice3/TestJouet.java]: You didn't choose the easiest path...
Compare the difference of actual and expected instead ;), or even simpler... Use assertEquals(expected, actual, delta)

**Pull request closed without merging**